src/main/server/conf/jwt_marketplace_check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+check_marketplace() {
+  INSTALL_DIRECTORY="/usr/local/antmedia"
+  REST_URL='http://localhost:5080/rest/v2/server-settings'
+
+  # Store original values
+  ORIGINAL_SERVER_JWT_CONTROL=$(grep "^server.jwtServerControlEnabled=" $INSTALL_DIRECTORY/conf/red5.properties)
+  ORIGINAL_SERVER_JWT_SECRET=$(grep "^server.jwtServerSecretKey=" $INSTALL_DIRECTORY/conf/red5.properties)
+
+  # JWT Secret Key
+  SECRET_KEY=$(openssl rand -base64 32 | head -c 32)
+
+  # Update red5.properties
+  sed -i "s^server.jwtServerControlEnabled=.*^server.jwtServerControlEnabled=true^" $INSTALL_DIRECTORY/conf/red5.properties
+  sed -i "s^server.jwtServerSecretKey=.*^server.jwtServerSecretKey=$SECRET_KEY^" $INSTALL_DIRECTORY/conf/red5.properties
+
+  # Restart antmedia
+  systemctl restart antmedia
```

Comment:
I think restarting server probably causes sync issues and more controls at this stage.